README.md

```diff
@@ -150,12 +150,12 @@ Once the `cloe-launch` tool is available, you can do one of the following:
  2. Launch `cloe-engine` directly:
     ```console
     $ cloe-launch -v exec -P conanfile.py -- usage
-    Source profile: tests/profile_default.py
+    Source profile: tests/conanfile_default.py
```

Comment:
What I understand is that you want to differentiate when you use a conanfile to build a package or to create a virtual conan environment and execute some command in it. If this is not like this, please tell me so because that means I'm not thinking this in the correct way.

In my opinion, calling it something different than conanfile could be misleading. They are conanfiles even if we use them for different purposes.

Reply:
We're on the same page here. Above I'm talking about `cloe-launch` wording "Source profile:", not the filename. But I suggest we do this in a separate PR.